File: 3412-permutation-difference-between-two-strings/3412-permutation-difference-between-two-strings.cs
public class Solution
{
   public int FindPermutationDifference(string s, string t)
    {
        List<int> differences = new List<int>();

        for (int i = 0; i < s.Length; i++)
        {
            for (int j = 0; j < t.Length; j++)
            {
                if (s[i] == t[j])
                {
                    differences.Add(i - j);
                }
            }
        }

     
        return differences.Sum(x => Math.Abs(x));
       

}
}

File: 3412-permutation-difference-between-two-strings/3412-permutation-difference-between-two-strings.js
/**
 * @param {string} s
 * @param {string} t
 * @return {number}
 */
var findPermutationDifference = function(s, t) {
    let arr = [];
    for (let i = 0; i < s.length; i++) {
        for (let j = 0; j < t.length; j++) {
            if (s[i] === t[j]) {
                arr.push([i-j]);
            }
        }
    }
    
    let res=arr.flat(Infinity).reduce((a,b)=>Math.abs(a)+Math.abs(b))
    return res;
};
